skyportal/handlers/api/photometry.py

```diff
@@ -751,7 +751,8 @@ def get_group_ids(data, user):
         )
 
     # always add the single user group
-    group_ids.append(user.single_user_group.id)
+    if hasattr(user, 'single_user_group'):
```

Comment:
under what circumstances does the user not have a single_user_group? Is it when it is a token? because I feel like that still requires us to add the single user group of the underlying user...

Reply:
yes when it is a token.

Reply:
Can we get the underlying user in that case?